tests/ckeditor/ckeditor.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect, beforeEach, document */
+
+'use strict';
+
+var modules = bender.amd.require( 'ckeditor', 'editor', 'promise' );
+
+var content = document.getElementById( content );
```

Comment:
You know... it's about parameters validation and this is something that we don't do on purpose, to make the code lighter. Otherwise we should start having validation on every method.

So, create() really doesn't care about what you pass to it... for now. At some point we'll start doing something with that element and by then things will not work if you pass something that the docs didn't tell you to pass.... like in any other method in the API.